src/components/cards/FeedbackModal.tsx

```diff
@@ -11,7 +11,7 @@ export const FeedbackModal = (props: Props) => {
 
   return (
     <React.Fragment>
-      {actionIconSchema[props.modalState.value].form(props.modalState)}
+      {actionIconSchema[props.modalState.value].form(props.modalState.reset)}
```

Comment:
Look into identifying these with enums, rather than index'. The number of buttons wont scale, so the clarity of that pattern might be desirable.

Reply:
Larger issue here really calls for some better "Enum State Machine" type pattern. Will look at this later.